lib/pbench/test/unit/agent/test_tool_meister.py

```diff
@@ -0,0 +1,153 @@
+"""
+Tests for the Tool Data Meister modules.
+"""
+import pathlib
+import subprocess
+import pytest
+
+from pbench.agent.tool_meister import ToolMeister, get_logger
+
+logger = get_logger("__logger__")
```

Comment:
I don't think a global logger should be used.  That creates state carried between tests.

Reply:
Still applies.